src/Scene/Node.js

```diff
@@ -31,6 +31,14 @@ define('Scene/Node', [], function() {
     }
 
 
+    Node.prototype.setVisibility = function(show) {
```

Comment:
I agree. I reused the same name as the other setVisibility functions, so the change should be reflected on all the other functions as well.